tests/System.Numerics.Tensors.Tests/TensorTests.cs

```diff
@@ -986,7 +1104,7 @@ public void Reshape(TensorConstructor tensorConstructor)
             };
 
             var tensor = tensorConstructor.CreateFromArray<int>(arr);
-            var actual = tensor.Reshape(3, 2);
+            var actual = tensor.Reshape(new[] { 3, 2 });
```

Comment:
BTW, do you know that you can now allocate spans on the stack in safe context? Maybe even the following would work:
```c#
var actual = tensor.Reshape(stackalloc int[] { 3, 2 });
```

cc: @VSadov

Reply:
Speaking of which: @VSadov / @jaredpar, are you guys still thinking about non allocating params Span<T>?

Reply:
Definitely thinking about it and something i want to do. Not convinced that we should use it Span directly. Been looking at a few ideas here.

Reply:
Actually, can you point me at the safe syntax that uses stackalloc and initializes inline?  I've tried a number of things and get messages that tells me the compiler doesn't know what I'm trying to do.

Reply:
I checked, the syntax does not work now, though it would be nice if it did.

Reply:
I couldn't find any syntax that works for stackalloc outside unsafe right now, not even https://github.com/dotnet/corefxlab/blob/master/docs/specs/span.md#language-support 4.

Reply:
There are numerous examples of this in corfxlab. See https://github.com/dotnet/corefxlab/blob/74ca8904031daf2d734104b90a6a5dadd22285c0/src/System.IO.Pipelines.Text.Primitives/ReadableBufferExtensions.cs#L115

Reply:
> I couldn't find any syntax that works for stackalloc outside unsafe right now, not even https://github.com/dotnet/corefxlab/blob/master/docs/specs/span.md#language-support 4.

We should update the span design doc to show the correct stackalloc syntax.
Change:
```C#
void SafeMethod() {
    var buffer = stackalloc Span<byte>(128);
    PrimitiveFormatter.TryFormat(buffer, DateTime.Now, ...);
}
```
to:
```C#
void SafeMethod() {
    Span<byte> buffer = stackalloc byte[128];
    PrimitiveFormatter.TryFormat(buffer, DateTime.Now, ...);
}
```

Reply:
Do you suppose there will be any syntax for inline initialization, or is that out of scope?

Reply:
@VSadov would have to answer this one.

Reply:
BTW, I updated the Span spec with the correct syntax.

Reply:
Thanks, I've updated the product code to use stackalloc where I can.  I'm not going to bother for these tests until we have good inline initializer syntax as I think it makes things less readable.

Reply:
Makes sense